strings/steve.json

```
@@ -15,6 +19,11 @@
         "We should store bank records in a webscale datastore, like /dev/null.",
         "You ever wonder if /dev/null supports sharding?",
         "Do you know who ate all the donuts?",
-        "What if we use a language that was written on a napkin and created over 1 weekend for all of our servers?"
+        "What if we use a language that was written on a napkin and created over 1 weekend for all of our servers?",
+        "Bruh what if we took a locker, some random trash and made a mech out of it?",
+        "Ok here me out, what if we make illegal, not illegal so sec stops arresting us for having it.",
+        "Crazy idea, ok ok ok, rather then having monkeys to test on, what if we only used apes.",
+        "Woh man ok, what if we took slime cores and smashed them into other slimes, be kinda cool to see what happens.",
+        "Were Nano-trasen but we need to unlock nano-parts whats the deal with that?"
```

Use apostrophes for shortening are into 're (we're) and is into 's (what's)
Nanotrasen not nano-trasen. Use caps to emphasize the Nano in nanotrasen.
Nano-parts should not be connected with a -. Nano is an adjective and parts is a noun.
Commas, don't forget your commas
```suggestion
        "We're NANOtrasen but we need to unlock nano parts, what's the deal with that?"
```